File: TransCoder_tokenized_test_set_functions/java/FIND_SUM_NON_REPEATING_DISTINCT_ELEMENTS_ARRAY.java
static int findSum ( int arr [ ], int n ) {
  int sum = 0;
  HashSet < Integer > s = new HashSet < Integer > ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( ! s . contains ( arr [ i ] ) ) {
      sum += arr [ i ];
      s . add ( arr [ i ] );
    }
  }
  return sum;
}

File: results/py/FIND_SUM_NON_REPEATING_DISTINCT_ELEMENTS_ARRAY.cpp
int f_filled ( int arr [ ] , int n ) {
  set < int > s ;
  int sum = 0 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( arr [ i ] != 0 ) s . insert ( arr [ i ] ) ;
  }
  for ( set < int > :: iterator it = s . begin ( ) ;
  it != s . end ( ) ;
  it ++ ) sum = sum + * it ;
  return sum ;
}
